### PR TITLE
[13.0][MIG] stock_account: generate Stock valuation layers

### PR DESCRIPTION
**Pull request by kos94ok-3D**
according to Stock move and Product price history

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kos94ok-3D** [comment]
@MiquelRForgeFlow It`s not finished but what do you think about it?

**Comment by MiquelRForgeFlow** [comment]
Thanks. I need some time to check, and I am these days occupied with other things, so don't expect I will review this PR soon. But good work. When you finish it, tell me again :)

**Comment by kos94ok-3D** [comment]
Ok

**Comment by kos94ok-3D** [comment]
@MiquelRForgeFlow, I finished

File: addons/stock_account/migrations/13.0.1.1/post-migration.py [hunk 1]
```diff
@@ -1,41 +1,328 @@
...
+    valued_move_lines = move.move_line_ids
+    valued_quantity = 0.0
+    for valued_move_line in valued_move_lines:
+        valued_quantity += valued_move_line.product_uom_id._compute_quantity(valued_move_line.qty_done,
+                                                                             move.product_id.uom_id)
+    quantity = valued_quantity
+
+    _change_standard_price(env, move, std_price_update)
+    unit_cost = _get_price_unit(move, std_price_update)
+    company = move.product.env.company
```
Comment (HviorForgeFlow) [inline addons/stock_account/migrations/13.0.1.1/post-migration.py]: I'm getting this error during migrations
```
2020-09-18 09:02:01,907 1 ERROR prod OpenUpgrade: stock_account: error in migration script stock_account/migrations/13.0.1.1/post-migration.py: 'stock.move' object has no attribute 'product' 
2020-09-18 09:02:01,907 1 ERROR prod OpenUpgrade: 'stock.move' object has no attribute 'product' 
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/openupgradelib/openupgrade.py", line 1795, in wrapped_function
    if use_env2 else cr, version)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 325, in migrate
    generate_stock_valuation_layer(env)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 32, in generate_stock_valuation_layer
    _create_dropshipped_svl(env, move, std_price_update)
  File "/opt/odoo/auto/addons/stock_account/migrations/13.0.1.1/post-migration.py", line 121, in _create_dropshipped_svl
    company = move.product.env.company
AttributeError: 'stock.move' object has no attribute 'product'
2020-09-18 09:02:01,910 1 ERROR prod odoo.modules.loading: Error executing post migration script for module stock_account
```

Comment (MiquelRForgeFlow) [reply addons/stock_account/migrations/13.0.1.1/post-migration.py:121]: try using https://github.com/ForgeFlow/OpenUpgrade/pull/17

Comment (kos94ok-3D) [reply addons/stock_account/migrations/13.0.1.1/post-migration.py]: Thanks for your response @HviorForgeFlow.
I fixed it. Please check again.

**Pull request merged**